numpy/core/tests/test_multiarray.py

```diff
@@ -9989,3 +9995,39 @@ def test_sort_uint():
 
 def test_private_get_ndarray_c_version():
     assert isinstance(_get_ndarray_c_version(), int)
+
+
+@pytest.mark.parametrize("N", np.arange(1, 512))
+@pytest.mark.parametrize("dtype", [np.float32, np.float64])
+def test_argsort_float(N, dtype):
+    rnd = np.random.RandomState(116112)
+    # (1) Regular data with a few nan: doesn't use vectorized sort
+    arr = -0.5 + rnd.rand(N).astype(dtype)
```

Comment:
`rand` is not the preferred alias.

```suggestion
    arr = -0.5 + rnd.random(N).astype(dtype)
```